js/msg.js

```diff
@@ -154,18 +160,23 @@
       const bg = getExtBg() ||
         chrome.tabs && await browser.runtime.getBackgroundPage().catch(() => {});
       const message = {method: 'invokeAPI', path, args};
-      let res;
       // content scripts, probably private tabs, and our extension tab during Chrome startup
       if (!bg) {
-        res = msg.send(message);
-      } else {
-        res = deepMerge(await bg.msg._execute(TARGETS.extension, message, {
+        return msg.send(message);
+      }
+      const err = new Error(`Callstack before invoking API.${path.join('.')}:`);
```

Comment:
It seems that we don't need this according to my screenshot?

Reply:
Your screenshot shows a local callstack after `Callstack before invoking API.styles.delete`, which is the entire point of this PR, so we need it.

Reply:
Before the patch, it already shows the correct stack:
![image](https://user-images.githubusercontent.com/1324510/146416385-16ee15b9-464a-4997-bfc2-5fcda5100ddd.png)
After the patch, it shows the same stack twice:
![image](https://user-images.githubusercontent.com/1324510/146416416-2b471778-9fd6-49c1-887e-1143891e7910.png)